src/viewport/viewport.js

```diff
@@ -79,14 +79,14 @@ export default class WebGLViewport extends Viewport {
       // Projection mode values
       projectionMode,
       projectionCenter,
-
       // Main projection matrices
       projectionMatrix: this.glProjectionMatrix,
       projectionFP64: this.glProjectionMatrixFP64,
       projectionPixelsPerUnit: this.pixelsPerMeter,
       projectionScale: this.scale,
       projectionScaleFP64: fp64ify(this.scale),
-
+      // viewMatrix is needed for backward-compatibility to deck.gl v2 shaders
+      viewMatrix: this.glViewMatrix,
```

Comment:
Should this go under the "deprecated" comment? As we continue to introduce shader modules, we might want to phase out support for this.